app/helpers/metric_item_helper.rb

```diff
@@ -12,12 +12,12 @@ def metric_item(metric_item, metric_value, sampled: false, html: {})
     html[:class] = Array.wrap(html[:class])
     html[:class] << 'sampled' if sampled
 
-    if item.display_guidance?
-      content += content_tag(:span, class: 'm-metric-guidance-toggle') do
-        id_text = "guidance-#{metric_item.identifier}"
-        content_tag(:a, '+', id: id_text, href: '#', class: 'a-metric-guidance-expand', data: { behaviour: 'a-metric-guidance-toggle' })
-      end
-    end
+    #if item.display_guidance?
+    #  content += content_tag(:span, class: 'm-metric-guidance-toggle') do
+    #    id_text = "guidance-#{metric_item.identifier}"
+    #    content_tag(:a, '+', id: id_text, href: '#', class: 'a-metric-guidance-expand', data: { behaviour: 'a-metric-guidance-toggle' })
+    #   end
+    # end
```

Comment:
Why has this been commented?

Reply:
Because this used to generate the + icon that expanded out an explanation.  I've put it under the metric title link now, but even that is wrong.